### PR TITLE
Finishing Analysis Work

### PR DESCRIPTION
**Pull request by corian-jal**
Tasks to Complete:
- Add functionality for pushing to the Google Drive
- Update all implementations to use the Google Drive and not local files
- Update the README with the data_analysis folder

### CODE REVIEW

**Comment by michplunkett** [comment]
I'm gonna get home tonight and I can take a look at it. After we merge this, if you could just add a sample of using it in the final Jupyter notebook you'd be all done!

**Comment by michplunkett** [comment]
And fantastic work! ☺️

File: data_analysis/night_record.py
Comment (michplunkett) [inline data_analysis/night_record.py]: Right here it looks like you aren't using the output from the morning file at all and are just running through the night file lines again.

**Review by michplunkett — CHANGES_REQUESTED** [review]

File: data_analysis/night_record.py
Comment (corian-jal) [reply data_analysis/night_record.py]: Fixed!

**Review by michplunkett — APPROVED** [review]

**Pull request merged**
